### ISSUE TITLE
Changes to EA Resources

### ISSUE THREAD

**Issue by Richenda** [rtcharity/hubreboot#105]
EA Resources looks like it does now because when we released it the rest of the Hub wasn't ready. Now that the main EA Hub is ready, we can tone down the Resources landing page. It needs the banner  image removed. It should only have the simple teal colour header with the EA Hub logo. The banner image we should reserve for the full Hub home page.

Also we want to introduce a hamburger menu that is the same everywhere on the Hub. Right now it only exists in the Resources page. We will make the 'About' page something that covers the whole EA Hub, not just the Resources site, and it should be in the sitewide drop down menu.

Site wide drop down menu should be:

- Sign up
- Log in
- Settings
- Profiles
- Groups
- Resources
- Map (once we work out how to take people to a segment of the home page. In the meantime I will sound out the rest of the team about having a dedicated map page or not)
- About the EA Hub

Finally, it would be great to have /Resources next to the EA Hub logo like with Manoj's first build.

**Comment by sebgrebe** [comment]
`Map (once we work out how to take people to a segment of the home page. In the meantime I will sound out the rest of the team about having a dedicated map page or not)`
@Richenda we've worked this out (wasn't tricky). So I don't think we need a dedicated map page.

**Comment by sebgrebe** [comment]
@richenda Done. You can see the results [here](https://confident-minsky-f713d6.netlify.com/#). This is not the actual resources page as I can't make this live before the eahub is live.

I had to tweak the menu a bit. To make sure that it looks analogous to the eahub (which I assumed was the highest priority), I didn't add anything to the main menu points that's not currently on the eahub. To make sure that we have navigation for the resources page though, I've added the sections of the resources page under "More".

Leave feedback as comments under here and tag me.

**Comment by sebgrebe** [comment]
Stuff on the navbar keeps changing, I will look at this again a last time just before launch.

**Comment by Richenda** [comment]
> @Richenda Done. You can see the results [here](https://confident-minsky-f713d6.netlify.com/#). This is not the actual resources page as I can't make this live before the eahub is live.
> 
> I had to tweak the menu a bit.
> * Consistent with what I explained [here](https://github.com/rtcharity/hubreboot/issues/137#issuecomment-457918538), **we're not going with the burger menu on desktop atm**

> Leave feedback as comments under here and tag me.
So we'll backpedal on this right? @sebgrebe

**Comment by sebgrebe** [comment]
> So we'll backpedal on this right? @sebgrebe

@Richenda Yes, is on my list :) I'll wait until just before launch, in case things should change again on the hub (which I don't expect though)

**Comment by Richenda** [comment]
> > So we'll backpedal on this right? @sebgrebe
> 
> @Richenda Yes, is on my list :) I'll wait until just before launch, in case things should change again on the hub (which I don't expect though)

Smart. Thank you!

**Comment by sebgrebe** [comment]
I don't expect any big changes before the soft launch, so I've fully updated EA Resources now. Result is [here](https://confident-minsky-f713d6.netlify.com/). Notice that this is a test page, not the actual resources domain, because that shouldn't be updated before the eahub goes fully love. I've also linked "EA Resources" on the hub to the test resources page for the moment, so that hub testers see what it should look like in the end.

@Richenda Feel free to leave feedback here.

**Comment by Richenda** [comment]
Looking great to me. Minor issue is that font size of 'popular articles' competes with the font size of the heading for that section. Perhaps making them 1 or 2 font sizes lower. Non urgent.
